Preliminaries problem overview (#6)
chhoumann/P6: sections/preliminaries.tex

Finally, the vector of attributes for the \textit{i}-th sensor at timestamp \textit{t} is represented with \(x_{t}^{(i)} \in \mathbb{R}^{C}\). \cite{cirsteaEnhanceNetPluginNeural2021}

% maybe TODO: Define correlated time series forecasting, if we end up using it.
% TODO: Formal problem definition 

\subsection{Problem overview}

> Given a time series as aforementioned, it is useful to be able to forecast the future behavior of the time series.

Reviewer: ```suggestion
Given a time series as previously mentioned, it is useful to be able to forecast the future behavior of the time series.
```

> This can be done by using a model that is able to predict the future behavior of the time series.

Reviewer: ```suggestion
This can be done using a statistical model that is able to predict that future behavior.
```

Author: Not sure statistical is the right word here. ARIMA is the only stats model we use.

Author: ```suggestion
This can be done using a model that is able to predict that future behavior.
```

> That is, we predict some target feature value at a future time step, e.g., the next value of the temperature.

Reviewer: ```suggestion
That is, we predict some target feature value at a future time step, for example, the next value of the temperature.
```

> There exist many techniques to do such forecasting. Some of these were presented in section \ref{sec:relatedwork}.

Reviewer: ```suggestion
There exist many different statistical models that can be used to perform time series forecasting. Some of these were presented in section \ref{sec:relatedwork}.
```

Author: Same as above

Author: ```suggestion
There exist many techniques that can be used to perform time series forecasting. Some of these were presented in section \ref{sec:relatedwork}.
```

> Given these models, there must be a way to compare their performance.

Reviewer: ```suggestion
A common approach to finding the best-performing model for a given task is to measure its performance on different data sets and compare it to other models.
```

> Therefore, we will compare the performance of the previously mentioned methods to the more recent transformer model.

Reviewer: ```suggestion
As such, we will compare the performance of the previously described models to the more recent transformer model.
```

> The task is to predict the mean temperature in a time series at a future time step, given data about previous time steps.

Reviewer: ```suggestion
The goal is to do univariate time series forecasting using the mean temperature of the different data sets included in the appendix. 
% TODO: Reference the appendix if we include the datasets there.
```

Author: ```suggestion
The goal is to predict the mean temperature in a time series at a future time step using data from preceding time steps.
```
